.gitignore

```diff
@@ -106,4 +106,4 @@ lib/**/*
 # Nix builds
 result
 
-src/package.json
+src/package.ts
```

Comment:
...this seems wrong?

Reply:
(There is no `src/package.json` _or_ `src/package.ts` file)

Reply:
Oh, I see, it's related to Luc's comment above...

Reply:
ya b/c it's .gitignored :)